### PR TITLE
Add CentOS base images

### PR DESCRIPTION
**Pull request by gozer**
Fixes #394 
Helps #154

### CODE REVIEW

File: nubis/Puppetfile [hunk 1]
```diff
@@ -46,14 +46,17 @@ mod 'nubis-confd_site',
     :git => 'https://github.com/gozer/nubis-site_confd.git',
     :ref => '0.1.1'
 
-mod 'srf/fluentd', '0.1.4'
+mod 'srf/fluentd',
```
Comment (limed) [inline nubis/Puppetfile]: Do these commits have anything to do with the centos support, or just a follow along commit?

File: nubis/Puppetfile [hunk 2]
```diff
@@ -46,14 +46,17 @@ mod 'nubis-confd_site',
...
+    :ref => 'd3b744bd591b2c44169f664f53f888ad4dc33041'
+
 mod 'mjhas/postfix', '1.0.0'
 
 #mod 'lex/dnsmasq','2.6.1'
 mod 'bhourigan/dnsmasq',
     :git => 'https://github.com/bhourigan/puppet-dnsmasq.git'
 
-mod 'datadog/datadog_agent', '1.6.0'
+mod 'datadog/datadog_agent', '1.7.1'
```
Comment (limed) [inline nubis/Puppetfile]: Same comment as above

File: nubis/Puppetfile (hunk 1 above)
Comment (gozer) [inline nubis/Puppetfile]: Yeah, needed to fix issues with the puppet modules and EL7

File: nubis/Puppetfile (hunk 2 above)
Comment (gozer) [inline nubis/Puppetfile]: Same answer, EL7 support

**Comment by limed** [comment]
r+

**Pull request merged**